griptapecli/core/skatepark.py

```diff
@@ -23,6 +25,8 @@
 
 state = State()
 
+DEFAULT_CLOUD_RUN_DELAY = "0"
```

Comment:
per above comment, suggest setting this to a value that reduces probability of it being skipped.